composer.json

```diff
@@ -34,55 +34,55 @@
         "friendsoftwig/twigcs": "^5.0",
         "gedmo/doctrine-extensions": "^3.0",
         "itk-dev/datafordeler-php-client": "dev-develop",
-        "itk-dev/openid-connect-bundle": "^1.0",
+        "itk-dev/openid-connect-bundle": "dev-feature/symfony-5.4",
         "knplabs/knp-paginator-bundle": "^5.6",
         "league/oauth2-client": "^2.6",
         "lexik/form-filter-bundle": "^7.0",
         "phpdocumentor/reflection-docblock": "^5.2",
         "sensio/framework-extra-bundle": "^5.1",
-        "symfony/asset": "5.2.*",
-        "symfony/console": "5.2.*",
-        "symfony/dotenv": "5.2.*",
-        "symfony/expression-language": "5.2.*",
+        "symfony/asset": "5.4.*",
+        "symfony/console": "5.4.*",
+        "symfony/dotenv": "5.4.*",
+        "symfony/expression-language": "5.4.*",
         "symfony/flex": "^1.3.1",
-        "symfony/form": "5.2.*",
-        "symfony/framework-bundle": "5.2.*",
-        "symfony/http-client": "5.2.*",
-        "symfony/intl": "5.2.*",
-        "symfony/lock": "5.2.*",
-        "symfony/mailer": "5.2.*",
-        "symfony/mime": "5.2.*",
+        "symfony/form": "5.4.*",
+        "symfony/framework-bundle": "5.4.*",
+        "symfony/http-client": "5.4.*",
+        "symfony/intl": "5.4.*",
+        "symfony/lock": "5.4.*",
+        "symfony/mailer": "5.4.*",
+        "symfony/mime": "5.4.*",
         "symfony/monolog-bundle": "^3.1",
-        "symfony/notifier": "5.2.*",
-        "symfony/process": "5.2.*",
-        "symfony/property-access": "5.2.*",
-        "symfony/property-info": "5.2.*",
-        "symfony/proxy-manager-bridge": "5.2.*",
-        "symfony/security-bundle": "5.2.*",
-        "symfony/serializer": "5.2.*",
-        "symfony/string": "5.2.*",
-        "symfony/translation": "5.2.*",
-        "symfony/twig-bundle": "^5.2",
-        "symfony/uid": "5.2.*",
-        "symfony/validator": "5.2.*",
-        "symfony/web-link": "5.2.*",
+        "symfony/notifier": "5.4.*",
+        "symfony/process": "5.4.*",
+        "symfony/property-access": "5.4.*",
+        "symfony/property-info": "5.4.*",
+        "symfony/proxy-manager-bridge": "5.4.*",
+        "symfony/security-bundle": "5.4.*",
+        "symfony/serializer": "5.4.*",
+        "symfony/string": "5.4.*",
+        "symfony/translation": "5.4.*",
+        "symfony/twig-bundle": "^5.3",
```

Comment:
Shouldn't this also be `5.4.*`?

Reply:
Yes. Fixed in https://github.com/itk-dev/naevnssekretariatet/pull/82/commits/886511f8eaf5f0e8e2861ef4d734d44010f4158f.